### PR TITLE
[Rancher2] Provide Windows Server version context for Rancher

### PR DESCRIPTION
**Pull request by nickgerace**
Relevant issue: https://github.com/rancher/docs/issues/2991

The motivation for the PR is in the issue above.

### CODE REVIEW

**Review by paynejacob — APPROVED** [review]

File: content/rancher/v2.x/en/cluster-provisioning/rke-clusters/windows-clusters/_index.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ The general node requirements for networking, operating systems, and Docker are
 In order to add Windows worker nodes to a cluster, the node must be running one of the following Windows Server versions and the corresponding version of Docker Engine - Enterprise Edition (EE):
 
 - Nodes with Windows Server core version 1809 should use Docker EE-basic 18.09 or Docker EE-basic 19.03.
-- Nodes with Windows Server core version 1903 should use Docker EE-basic 19.03.
+- Nodes with Windows Server core version 1903+ should use Docker EE-basic 19.03.
```
Comment (aiyengar2) [inline content/rancher/v2.x/en/cluster-provisioning/rke-clusters/windows-clusters/_index.md:41]:
```suggestion
- Nodes with Windows Server Core version 1903+ should use Basic Docker EE 19.03.
```

We should follow the official naming scheme in the docs

File: content/rancher/v2.x/en/cluster-provisioning/rke-clusters/windows-clusters/_index.md [hunk 2]
```diff
@@ -48,6 +48,12 @@ In order to add Windows worker nodes to a cluster, the node must be running one
 ### Kubernetes Version
 
 Kubernetes v1.15+ is required.
+The [Kubernetes docs for Windows nodes](https://kubernetes.io/docs/setup/production-environment/windows/intro-windows-in-kubernetes/#windows-os-version-support) provide a compatability matrix for which versions of Windows Server will be compatible with your version of Kubernetes.
```
Comment (aiyengar2) [inline content/rancher/v2.x/en/cluster-provisioning/rke-clusters/windows-clusters/_index.md:51]:
```suggestion
The [Kubernetes docs for Windows nodes](https://kubernetes.io/docs/setup/production-environment/windows/intro-windows-in-kubernetes/#windows-os-version-support) provide a compatibility matrix that can be used to assess which versions of Windows Server are supported based on your Kubernetes cluster version.
```

File: content/rancher/v2.x/en/cluster-provisioning/rke-clusters/windows-clusters/_index.md [hunk 3]
```diff
@@ -48,6 +48,12 @@ In order to add Windows worker nodes to a cluster, the node must be running one
...
+
+> **More Information on Windows Server Versions:**
+>
+> - [Windows Server release information](https://docs.microsoft.com/en-us/windows-server/get-started/windows-server-release-info)
+> - [Windows Server servicing channels: LTSC and SAC](https://docs.microsoft.com/en-us/windows-server/get-started-19/servicing-channels-19)
```
Comment (aiyengar2) [inline content/rancher/v2.x/en/cluster-provisioning/rke-clusters/windows-clusters/_index.md:56]:
```suggestion
> - [Release Information](https://docs.microsoft.com/en-us/windows-server/get-started/windows-server-release-info)
> - [Servicing Channels: LTSC and SAC](https://docs.microsoft.com/en-us/windows-server/get-started-19/servicing-channels-19)
```

Windows Server is redundant

File: content/rancher/v2.x/en/cluster-provisioning/rke-clusters/windows-clusters/_index.md [hunk 4]
```diff
@@ -239,4 +245,4 @@ After creating your cluster, you can access it through the Rancher UI. As a best
 
 # Configuration for Storage Classes in Azure
 
-If you are using Azure VMs for your nodes, you can use [Azure files](https://docs.microsoft.com/en-us/azure/aks/azure-files-dynamic-pv) as a [storage class]({{<baseurl>}}/rancher/v2.x/en/cluster-admin/volumes-and-storage/#adding-storage-classes) for the cluster. For details, refer to [this section.]({{<baseurl>}}/rancher/v2.x/en/cluster-provisioning/rke-clusters/windows-clusters/azure-storageclass)
\ No newline at end of file
+If you are using Azure VMs for your nodes, you can use [Azure files](https://docs.microsoft.com/en-us/azure/aks/azure-files-dynamic-pv) as a [storage class]({{<baseurl>}}/rancher/v2.x/en/cluster-admin/volumes-and-storage/#adding-storage-classes) for the cluster. For details, refer to [this section.]({{<baseurl>}}/rancher/v2.x/en/cluster-provisioning/rke-clusters/windows-clusters/azure-storageclass)
```
Comment (aiyengar2) [inline content/rancher/v2.x/en/cluster-provisioning/rke-clusters/windows-clusters/_index.md:248]:
```suggestion
If you are using Azure VMs for your nodes, you can use [Azure files](https://docs.microsoft.com/en-us/azure/aks/azure-files-dynamic-pv) as a [storage class]({{<baseurl>}}/rancher/v2.x/en/cluster-admin/volumes-and-storage/#adding-storage-classes) that is available on your cluster. For details, refer to [this section.]({{<baseurl>}}/rancher/v2.x/en/cluster-provisioning/rke-clusters/windows-clusters/azure-storageclass)
```

either `the storage class for the cluster` or `a storage class that is available on your cluster`

**Review by aiyengar2 — CHANGES_REQUESTED** [review]
LGTM

File: content/rancher/v2.x/en/cluster-provisioning/rke-clusters/windows-clusters/_index.md (hunk 1 above)
Comment (nickgerace) [reply content/rancher/v2.x/en/cluster-provisioning/rke-clusters/windows-clusters/_index.md:41]: Great observation. I mainly tried to keep things similar, but it's good to re-evaluate.

File: content/rancher/v2.x/en/cluster-provisioning/rke-clusters/windows-clusters/_index.md (hunk 2 above)
Comment (nickgerace) [reply content/rancher/v2.x/en/cluster-provisioning/rke-clusters/windows-clusters/_index.md:51]: Perfect. The liberal arts student in me is unhappy with myself right now. This is awesome :)

File: content/rancher/v2.x/en/cluster-provisioning/rke-clusters/windows-clusters/_index.md (hunk 3 above)
Comment (nickgerace) [reply content/rancher/v2.x/en/cluster-provisioning/rke-clusters/windows-clusters/_index.md:56]: This was a copy/paste from another place. Totally agree.

File: content/rancher/v2.x/en/cluster-provisioning/rke-clusters/windows-clusters/_index.md (hunk 4 above)
Comment (nickgerace) [reply content/rancher/v2.x/en/cluster-provisioning/rke-clusters/windows-clusters/_index.md:248]: Your change looks good!

Comment (nickgerace) [reply content/rancher/v2.x/en/cluster-provisioning/rke-clusters/windows-clusters/_index.md]: There was an upstream change that I rebased to that changed this a little. Fixed regardless.

**Comment by nickgerace** [comment]
Fixed @aiyengar2's nits.

**Review by aiyengar2 — APPROVED** [review]
LGTM

**Review by cbron — APPROVED** [review]

**Pull request merged**